packages/ifpack2/src/Ifpack2_ConfigDefs.hpp

```diff
@@ -77,11 +77,12 @@ namespace Ifpack2 {
     ///   enum may disappear or change name, or the values may change
     ///   names or values or disappear, at any time.
     enum RelaxationType {
-      JACOBI, //!< Jacobi
-      GS,     //!< Gauss-Seidel
-      SGS,    //!< Symmetric Gauss-Seidel
-      MTGS,   //!< Multicore Gauss-Seidel
-      MTSGS   //!< Multicore Symmetric Gauss-Seidel
+      JACOBI,  //!< Jacobi
+      GS,      //!< Gauss-Seidel
+      SGS,     //!< Symmetric Gauss-Seidel
+      MTGS,    //!< Multicore Gauss-Seidel
+      MTSGS,   //!< Multicore Symmetric Gauss-Seidel
+      MTJACOBI //!< Multicore Jacobi
```

Comment:
@mhoemmen: The implementation is different. "Jacobi", the old option, does A*x instead of R*x (A = D + R), for example. Also, BlockRelaxation does a serial pass over Container::apply calls. Thus, the old method is supported by all containers. The new method asks the Container to do the entire Jacobi iteration, which lets the Container do clever things. For this reason, the new method has an impl only in BlockTriDiContainer. Thus, we need to distinguish between them somehow. Open to suggestions.